spk/python310/Makefile

```diff
@@ -30,7 +30,7 @@ SPK_USR_LOCAL_LINKS = bin:bin/python3.10
 WHEELS = src/requirements-crossenv.txt src/requirements-pure.txt src/requirements-abi3.txt
 
 # Force building pure-python wheels
-#WHEELS_PURE_PYTHON_PACKAGING_ENABLE = 1
+WHEELS_PURE_PYTHON_PACKAGING_ENABLE = 1
```

Comment:
Why is this sometimes on and sometimes off?

Reply:
By default it's always off.  This mean that at package install `pip` will install all local wheels and download the missing ones.  Provided wheels are the cross-compileds ones while the pure-python which are arch agnostic are being downloaded from pypi with pip.  This allow reducing greatly the package size considering we're short on disk space.

When I enable it is to demonstrate that the framework still works as expected and that it can build pure-python wheels and package them.  Once all lights are green I re-disable them.